### DISCUSSION TITLE
Unauthorized - You cannot view this page. Login as...

### DISCUSSION THREAD

**Discussion by Wikojs** [requarks/wiki#5127] [Error / Bug Report]
In my group "Developers" I have following Redirect on Login:
/en/Developers/home 

Page Rules:
Path starts with /en/Developers

Why does the user who is in my "Developers" group gets notice;
Unauthorized - You cannot view this page. Login as...

please help 🥺

**Comment by kontr0x** [comment]
Does the error appear when trying to log in?

If so, make sure the group "Developers" has permission to see the login page.

**Reply by Wikojs** [reply]
Hi, thank you very much for the fast response !
 Yes, the error does happen when user tries to log in.

In the group "Developers" there is a Page Rule;
Path starts with /en/Developers
which should be saying that the group has the permission to see the login page...

If I can't get this permission functioning we won't be able to use Wiki JS in our firm, which would be a shame, because a lot of people here find it really cool :)


![wiki 1](https://user-images.githubusercontent.com/102657017/161022728-9164887b-0dde-48e1-935e-6660cbc43adf.png)

![wiki 2](https://user-images.githubusercontent.com/102657017/161022745-e229984b-d12e-4f05-816f-dbf8b1d74dbe.png)

**Reply by kontr0x** [reply]
Just to be clear, I'm not a developer or any contributor of Wiki.js!

I'm just here to help :)

I don't think you've covered the login page in the page rules.
Also, "/en/" is not needed in the path, that's covered with the locales settings.

My approach would be the following:

Start with the page rule where you start with the root directory ("/") and grant all permissions, then check if it works.

If it does, then get more and more specific with the path until you get it right.

**Reply by Wikojs** [reply]
Thank you for you help! I really just needed to create an extra login home page for every department.

What confused me is Wiki JS folder structure; I can create a complex path, creating a site, for example: "/developers/home/database/database2"
even though, "/developers/home" in this case does not exist as a callable page, if I dont create that page extra :`)

**Reply by kontr0x** [reply]
I'm glad that I could help you out.

Please make sure that you post a valid solution, for others that might run in the same issue.
Screenshots of the configuration are welcome. Make sure that you don't post confidential information.

**Comment by Wikojs** [comment]
With this permissions, it seems to work:

![dev new](https://user-images.githubusercontent.com/102657017/161767543-33141b40-2b3a-4e48-83d6-1daa135aa793.png)

![dev new 2](https://user-images.githubusercontent.com/102657017/161767574-3eda3b52-a7c3-4fd9-90f5-2b9b3d82b934.png)
